test/apiv2/python/conftest.py

```diff
@@ -0,0 +1,8 @@
+"""
+Configure pytest
+"""
+
+
+def pytest_report_header(config):
+    """Add header to report."""
+    return "python client -- requests library"
```

Comment:
I think `requests library` is an implementation detail.  For consistency, would it be more helpful to say `python client -- PodmanClient' (or similar)?

Ref: https://github.com/containers/podman/pull/13380/files#diff-6281c15876f0156ea34c5870ac0309f169d9dd47767dc30d9095210bc776700dR8

Reply:
@cevich These tests are driven from the requests library not PodmanClient or DockerClient.